src/postprocess/phonon.jl

```diff
@@ -36,3 +106,21 @@ function compute_dynmat_cart(basis::PlaneWaveBasis, ψ, occupation; kwargs...)
     dynmats_reduced = compute_dynmat(basis, ψ, occupation; kwargs...)
     dynmat_red_to_cart(basis.model, dynmats_reduced)
 end
+
+function compute_dynmat(scfres::NamedTuple; kwargs...)
+    compute_dynmat(scfres.basis, scfres.ψ, scfres.occupation; scfres.ρ, scfres.ham,
+                   scfres.occupation_threshold, scfres.εF, scfres.eigenvalues, kwargs...)
+end
+
+function compute_dynmat_cart(scfres; kwargs...)
```

Comment:
weird, can't you just have a blanket `compute_dynmat_cart(args...; kwargs...) = convert...` ?